app/grandchallenge/evaluation/forms.py

```diff
@@ -985,3 +985,25 @@ def clean_ground_truth(self):
             raise ValidationError("Ground truth updating already in progress.")
 
         return ground_truth
+
+
+class AlgorithmInterfaceForPhaseCopyForm(Form):
+    phases = ModelMultipleChoiceField(
+        queryset=Phase.objects.none(),
+        label="Select the phases to copy the interfaces to",
+        widget=CheckboxSelectMultiple,
+    )
+
+    def __init__(self, *args, phase, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._phase = phase
+
+        self.fields["phases"].queryset = Phase.objects.filter(
+            challenge=phase.challenge,
+            submission_kind=phase.submission_kind,
```

Comment:
Since we only allow interfaces for algorithm submission phases, you can subset by that directly, right?